### ISSUE TITLE
Provide linac simulation 

### ISSUE THREAD

**Issue by piogor** [3Controls/tangobox#9]
- [x] create a container for demos
- [x] install elinac device servers on the container

**Comment by piogor** [comment]
Please check with Sergi if the elinac demo from old tangobox is the same as the python one:
https://github.com/tango-controls/VACCA/tree/master/examples/elinac

**Comment by Patfra** [comment]
Sergi at  https://github.com/tango-controls/VACCA/tree/master/examples/elinac  writes that elinac requires Pytango and Taurus. The base image of container created by @stanislaw47 in #5 doesnt contains Taurus, but the VM eventually will contain Taurus (#14). As far as I am concerned, container with elinac will be running on VM and is supose to comunicate with it, so here is my question: Do I have to install Taurus on container? 

EDIT:
I tried to set up container without Taurus, but it cannot be done. This command in ipython cannot be executed without taurus: 
```
import SimulatorDS.gen_simulation as gs
```  
In this case I will install Taurus on the container.

**Comment by Patfra** [comment]
Greetings @sergirubio,
We have been working on new Tangobox VM and I have a question considering elinac. Do you know if elinac demo from old tangobox VM is the same as this on your [repository](https://github.com/tango-controls/VACCA/tree/master/examples/elinac)?

**Comment by sergirubio** [comment]
Hi, 

It's strange that you need taurus for importing SimulatorDS ; I'll crosscheck the dependencies to correct that.

The elinac simulation provided on Vacca repository is a very simplified version of the old tangobox demo; it provides the same attributes but no commands/states behavior.

I just included it to allow to test the Vacca application in few steps without extra dependencies; it can be used as minimal test.

Sergi

**Comment by Patfra** [comment]
@piogor I have managed to install elinac on container. ``tangobox-sim`` ?

However, I can not open the elinac example, because of this error:

![image](https://user-images.githubusercontent.com/11886864/30733849-01453f1e-9f79-11e7-8ce6-0f9971c18e4f.png)

I installed nano and used it to change files: /home/dservers/vacca.git/vacca/panel.py and /home/dservers/vacca.git/vacca/panel.py, but still there is an error:

![image](https://user-images.githubusercontent.com/11886864/30735500-8ddf838e-9f7f-11e7-97c9-e49c59560940.png)

@sergirubio do you know solution to this matter? Do I have to install PyTangoArchiving?

**Comment by sergirubio** [comment]
Ok, it's a problem with the last version of Taurus, I'll commit a patch to vacca sources.
PyTangoArchiving is optional but provides archiving plotting an attribute searches widgets to Taurus.

**Comment by sergirubio** [comment]
Latest sources have been pushed to master branch at github and pip

https://pypi.python.org/pypi/Vacca
https://github.com/tango-controls/VACCA/archive/VACCA_4.5.6.tar.gz

**Comment by sergirubio** [comment]
Get also the latest fandango

https://pypi.python.org/pypi/fandango
https://github.com/tango-controls/fandango/releases

**Comment by sergirubio** [comment]
I also uploaded latest Panic (6.3) and PyTangoArchiving (7.4) releases to pipy and github

**Comment by Patfra** [comment]
Thank you kindly @sergirubio :)
I will download them right away. I will give you feedback after I finished configuration and testing.

**Comment by Patfra** [comment]
I used new version of fatango and vacca, but there was error with importing:
```python
fandango.qt.Dropable()
```
in vacca/panel.py. I have checked importing files in python using ipython and discovered that to access to it I have to import not only fandango, but also fandango.qt. That fixed the error and i could run ``vaccagui elinac.py`` .  I have seen that devices in DserverDS are changing state (ON, ALARM, MOVING). Should it be like this?
